### DISCUSSION TITLE
SNAP 9.0 compatibility

### DISCUSSION THREAD

**Discussion by pbrotoisworo** [johntruckenbrodt/pyroSAR#212] [Q&A] [answered]
Given some of the major changes in SNAP 9, is pyroSAR compatible? I would assume not. Asking before I upgrade on my personal computer.

https://forum.step.esa.int/t/snap-9-0-released/36558

**Comment by johntruckenbrodt** [comment, marked as answer]
Hi @pbrotoisworo. SNAP 9 is not yet supported.

**Reply by johntruckenbrodt** [reply]
Update: pyroSAR 0.18.0 is now released, which works with SNAP 9.
